module-infrastructure/persistence-database/src/main/java/com/depromeet/friend/repository/FriendJpaRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.depromeet.friend.repository;
+
+import com.depromeet.friend.entity.FriendEntity;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface FriendJpaRepository extends JpaRepository<FriendEntity, Long> {
+    @Query(
+            "SELECT f "
+                    + "From FriendEntity f "
+                    + "WHERE f.member.id = :memberId "
+                    + "AND f.following.id = :followingId")
+    Optional<FriendEntity> findByMemberAndFollowing(
```

Comment:
```suggestion
    Optional<FriendEntity> findByMemberIdAndFollowingId(
```
메서드 이름을 이렇게 사용하지 않은 이유가 무엇인까요?

Reply:
쿼리DSL 로 변경해서 이 메서드는 이제 없습니다